webapp/portlet/src/main/webapp/vue-apps/people-list/initComponents.js

```diff
@@ -12,12 +15,17 @@ const components = {
   'people-card': PeopleCard,
   'people-card-front': PeopleCardFront,
   'people-card-reverse': PeopleCardReverse,
+  'people-advanced-filter-drawer': PeopleAdvancedFilterDrawer,
 };
 
 for (const key in components) {
   Vue.component(key, components[key]);
 }
-
+if (!Vue.prototype.$profileSettingsService) {
+  window.Object.defineProperty(Vue.prototype, '$profileSettingsService', {
+    value: ProfileSettingsService,
+  });
+}
```

Comment:
Same remark as previews, to delete and move to `commonVueComponent`

```suggestion
```

Reply:
yes already mentioned [here](https://github.com/Meeds-io/social/pull/2191#issuecomment-1410424867), it is declared like that since in the other MIP it isn't yet a common component, so it should be updated there before.

Reply:
fixed by this commit  https://github.com/Meeds-io/social/commit/616cd490ff344bc114a19d5b9f9163c1f4bb58a5

Reply:
fixed